ls-ui-react-app/src/pages/UploadCircuitPage.tsx

```diff
@@ -185,6 +196,7 @@ const UploadCircuitPage = ( {appState, setAppState} : AppStateProps)  =>
             <section>
                 <div>
                     <CompilerInputCircuitSelection appState={appState} setAppState={setAppState} />
+                    { appState.compilationIsLoading && <b>Loading ... </b> }
```

Comment:
@Keelando . When the `appState.compilationIsLoading` gets set to true, the symbol appears instantly. If we want an animation to make it appear slowly it's possible to do it using [css transitions](https://www.w3schools.com/css/css3_transitions.asp).

Though thinking about it's most likely not worth it :sweat_smile:, I still wanted to share for future reference

Reply:
Once things are set up with the desired basic functionality, ie correct lattice view rendering and style, I am happy to go to town with the niceties like animations, smooth scrolling into view, etc.